src/helpers/feed-helpers.ts

```diff
@@ -16,7 +17,7 @@ export async function submitFeedHelper(
   feedsApiClient: FeedsApiClientV20210630,
   feedType: string,
   content: string,
-  amazonMarketplaceId = 'ATVPDKIKX0DER',
+  amazonMarketplaceId = AMAZON_US_MARKETPLACE_ID,
```

Comment:
I guess it's for default purposes, is this specific to Amazon or specific to our Amazon account?

Reply:
This is amazon US wide, nothing specific about our account